### ISSUE TITLE
2.2.7 - Browser, Implementors of selected method not available any more

### ISSUE THREAD

**Issue by feldti** [jgfoster/Jade.bak#173]
If I am in the source code tab of a method (within a browser) and select a method (in that source code) I was able to get a pop-up with menu entries like "implementors" or "senders" and this is gone ?????!!!!!

**Comment by jgfoster** [comment]
What GemStone version, class, method, and selector is selected when you see this problem?
 
I believe that the context menu offers implementors/senders for most selectors. The exception is optimized selectors. For example, in GS 3.4.2, AbstractDictionary>>addAll: the selectors #isKindOf: (step point 2) and #ifTrue:ifFalse: (step point 3) do not offer implementors/senders, but #accompaniedBy:keysAndValuesDo: (step point 4) does offer implementors/senders.

I've requested the stepPoint->selector information from GsNMethod (internal request #43830).

**Comment by feldti** [comment]
I am still having this issue (Jade 2.2.7 against GS 3.4.0) - and it always works in the Method List Browser, but not in the System Browse. The menu entries simply do not appear - but it does not seem to be that sample.
When I switched to the Method List Browser I noticed, that the menu appears, then switched back to System browse and then suddenly the menues appears there also - and after a restart (of jade) it was still there ... something is wrong there, but do not know what.
